debian/control

```diff
@@ -34,10 +34,11 @@ Build-Depends: bison,
                texinfo (>= 4.7),
                lua5.3 <pkg.frr.lua>,
                liblua5.3-dev <pkg.frr.lua>,
-	       libgrpc-dev (>=1.16.1) <pkg.frr.grpc>,
-	       libgrpc++-dev (>=1.16.1) <pkg.frr.grpc>,
-	       protobuf-compiler (>=3.6.1) <pkg.frr.grpc>,
-	       protobuf-compiler-grpc (>=1.16.1) <pkg.frr.grpc>
+               libgrpc-dev (>=1.16.1) <pkg.frr.grpc>,
```

Comment:
please remove the unnecessary whitespace changes

Reply:
OK. Now the file looks like below, but as you want.

![image](https://github.com/user-attachments/assets/138a09c2-3d52-4b42-a057-7fd168adbb2c)